tools/ReleaseTool/ReleaseCommand.cs

```diff
@@ -174,12 +181,12 @@ private static string GetReleaseNotesFromChangeLog()
         {
             if (!File.Exists(ChangeLogFilename))
             {
-                Console.WriteLine("Could not get draft release notes, as the change log file, " +
-                                  $"{ChangeLogFilename}, does not exist.");
+                Logger.Warn("Could not get draft release notes, as the change log file, " +
```

Comment:
Hmmm should this be an error rather than a warning? This is if the `CHANGELOG.md` file actually doesn't exist right?

Reply:
Changed! :)